lib/ddio.h

```diff
@@ -320,14 +321,25 @@ void ddio_MouseSetVCoords(int width, int height);
 //	File Operations
 //	---------------------------------------------------------------------------
 
-//	creates or destroys a directory or folder on disk
-//	This pathname is *RELATIVE* not fully qualified
-bool ddio_CreateDir(const char *path);
-bool ddio_RemoveDir(const char *path);
+//	creates a directory on disk
+inline bool ddio_CreateDir(std::filesystem::path const& path) {
+  std::error_code ec{};
+  std::filesystem::create_directory(path, ec);
```

Comment:
Better to use `create_directories` instead. It's more universal, and we don't have to care about recursive creation of subdirectories.

Reply:
good idea